ompi/runtime/ompi_mpi_params.c

```diff
@@ -76,7 +76,13 @@ uint32_t ompi_add_procs_cutoff = OMPI_ADD_PROCS_CUTOFF_DEFAULT;
 bool ompi_mpi_dynamics_enabled = true;
 
 char *ompi_mpi_spc_attach_string = NULL;
```

Comment:
Can we protect these so that they dont appear in the library if SPC is not enabled ?